test/fixtures/output/kafka/17.golden

```diff
@@ -6,7 +6,7 @@
 | Ingress Limit (MB/s) | 250                       |
 | Egress Limit (MB/s)  | 750                       |
 | Storage              | 5 TB                      |
-| Provider             | aws                       |
+| Cloud                | aws                       |
```

Comment:
It's unfortunate that this API doesn't uppercase "aws" like our other APIs at the company... maybe they'll fix that in their next major update.